php/commands/rewrite.php

```diff
@@ -173,10 +173,22 @@ public function structure( $args, $assoc_args ) {
 	 * : Show rewrite rules from a particular source.
 	 *
 	 * [--fields=<fields>]
-	 * : Limit the output to specific fields. Defaults to match,query,source.
+	 * : Limit the output to specific fields.
+	 * ---
+	 * default: match,query,source
```

Comment:
I don't have any better idea, honestly. Three keywords are there. If we keep available options for that, we would have 6 combination from those three keywords.